core/client/set.go

```diff
@@ -324,6 +326,7 @@ func InitSDK(walletJSON string,
 		return err
 	}
 	SetSdkInitialized(true)
+	node.SetIsWasm(true)
```

Comment:
We can't force it for everyone using gosdk. 

Set it in wasm initialisation and mobile initialisation (we need to use it on mobile as well, perhaps change the name to something resonable).

Reply:
This InitSdk method is being called from both sdks only.

Reply:
But it is called from other repos using gosdk as well. We need to get it in input. Gosdk is used in 0box, blobber, CLIs and 0chain. Everywhere it can't use 0box.